src/picknik_ur_base_config/objectives/move_to_waypoint.xml

```diff
@@ -11,6 +11,11 @@
     </Control>
   </BehaviorTree>
   <TreeNodesModel>
-    <SubTree ID="Move to Waypoint"/>
+    <SubTree ID="Move to Waypoint">
+      <input_port name="waypoint_name" default=""/>
+      <input_port name="joint_group_name" default="manipulator"/>
+      <input_port name="use_all_planners" default="false"/>
```

Comment:
I think this option can be removed, since it is not used?

Reply:
Good catch, I initially did this in `v5.3`, but it has changed now in `main`. Updating to set `planner_interface` instead.